README.rst

```diff
@@ -9,7 +9,10 @@ make creating exploratory map views effortless. Here's a crash course:
 ::
 
     from gmplot import gmplot
-
+    
+    # Add Google API Key
+    gmap.apikey = "Enter_API_Key_Here"
+    
     # Place map
     gmap = gmplot.GoogleMapPlotter(37.766956, -122.438481, 13)
```

Comment:
`gmap.apikey` is accessed before `gmap` exists? Maybe the API key can be defined as a separate variable first, and then passed into `GoogleMapPlotter`?:

```
# Add Google API Key
apikey = "Enter_API_Key_Here"

# Place map
gmap = gmplot.GoogleMapPlotter(37.766956, -122.438481, 13, apikey)
```